app/start-server.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
```

Comment:
do we need the shabang? 
I think the intention is to run it via node using package.json start script... so probably its redundant